lib/parser.mly

```diff
@@ -44,12 +44,15 @@
 
 (* ---------------------------------------- *)
 %start <Syntax.scr_module> scr_module
-%{ open Syntax
+%{ module S = Syntax
```

Comment:
Why don't you `open S`

Reply:
MMMM, I don't have a good answer now, but while I was doing it, it helped separate Syntax.name from the parser name.

Reply:
I see, but now it's ok to open S?

Reply:
I ended up opening Syntax